plugins/animate/animate.cpp

```diff
@@ -243,13 +292,78 @@ class wayfire_animation : public wf::singleton_plugin_t<animation_global_cleanup
         return {"none", 0};
     }
 
+    bool reversed(wayfire_view view, wf_animation_type type, std::string name,
```

Comment:
IMO it would be a bit clearer if the function was called `try_reverse` and the direction which is given is the desired direction (currently, it is the 'wrong' direction)